apps/cms/contacts/models.py

```diff
@@ -32,15 +34,22 @@ class FormField(AbstractFormField):
 class WagtailCaptchaFormBuilder(FormBuilder):
     @property
     def formfields(self):
+        contacts = self.get_contacts_for_default_site()
         # Add captcha to formfields property
         fields = super().formfields
         fields['captcha'] = CaptcheckCaptchaField(
             label=_('I am not a robot'),
             help_text=_('If you are having difficulty please contact us, '
-                        'details can be found adjacent.'))
+                        'details can be found adjacent.') + contacts)
```

Comment:
Yeah, so this is the richtext as str.. not sure if that helps?

Reply:
ooooo, danke i'll give it a go

Reply:
@Rineee dumb question to add this to the signup form I just change self to form or do I need to restructure the class?

Reply:
NOTE: this solution retains all the html tags, would need to be cleaned first